src/chains/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+
+class Chain(models.Model):
+    id = models.CharField(primary_key=True, max_length=255)
```

Comment:
Why a CharField?

Reply:
The integer is constrained to 4 bytes. I don't think chain ids have this limit though so I went with a string representation.

Reply:
I would go with either a `IntegerField`, or if you are concerned about the size `Uint256Field`

`from gnosis.eth.django.models import EthereumAddressField, Uint256Field`

Reply:
Since this affects the model (introduced before pagination) I will tackle it in a different PR. Is that ok? 🙂